package.json

```diff
@@ -33,7 +33,7 @@
   "scripts": {
     "docs": "node node_modules/.bin/repo-tools exec -- jsdoc -c .jsdoc.js",
     "generate-scaffolding": "node node_modules/.bin/repo-tools generate all && node node_modules/.bin/repo-tools generate lib_samples_readme -l samples/ --config ../.cloud-repo-tools.json",
-    "lint": "node node_modules/.bin/repo-tools lint --cmd eslint -- build/src/ samples/ build/system-test/ build/test/",
+    "lint": "node node_modules/.bin/repo-tools lint --cmd eslint -- src/ samples/ system-test/ test/",
```

Comment:
We have .js files only in the samples/ directory, don't we?

Reply:
Does `eslint` support typescript at all? For me it looks like it does not, at least without some plugin. I guess it's not the right way to fix linting...

Reply:
My thought was that if we needed to add any Javascript files to the other directories (`src`, `test`, etc.) then they will be linted.  I think this question of how to do linting should be handled is outside the scope of this PR.  I'll make an issue to address it.